### PR TITLE
Release 3.2.4

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.2.4 (2019-05-28)
+
+- Fixed the list toolbar component to properly show list toolbar items which are dynamically shown and hidden. [#45](https://github.com/blackbaud/skyux-list-builder/pull/45)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: I don't follow this changelog entry as written. I **think** we just have "which" in the wrong place. Did we mean to have "which" after "show" and before "list tooblar"? i.c., "Fixed the list toolbar component to properly show which list toolbar items are dynamically shown and hidden"?

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: No. Basically if a consumer dynamically adds or removes a toolbar item in their code via an `ngIf` the current toolbar does not remove that item. So when the item is "dynamically shown or hidden" the toolbar doesn't update.

Comment (blackbaud-johnly) [reply CHANGELOG.md]: So would "Fixed the list toolbar component to properly display items when they are dynamically shown or hidden." work?

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: That's fine

Comment (blackbaud-johnly) [reply CHANGELOG.md]: Done.

**Pull request merged**